.rubocop.yml

```diff
@@ -45,10 +45,10 @@ Layout/LineLength:
 
 Metrics/AbcSize:
   Enabled: true
-  Max: 47
+  Max: 48
 
 Metrics/CyclomaticComplexity:
-  Max: 7
+  Max: 9
```

Comment:
xD gunna love watching this # creeeeeeeep up slowly

Reply:
Hah, same.